x/docs/client/cli/query.go

```diff
@@ -117,3 +121,33 @@ func GetCmdSharedDocuments(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// ----------------------------------
+// --- DocumentReceipt
+// ----------------------------------
+
+func GetCmdReceivedReceipts(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "receipts [user-address] [doc-uuid]",
```

Comment:
I think we should specify that the second parameter is optional, by wrapping it with brackets
```suggestion
		Use:   "receipts [user-address] [[doc-uuid]]",
```